SITnett/templates/registration/password_reset_email.html

```diff
@@ -0,0 +1,4 @@
+Noen ba om å tilbakestille passordet for kontoen med e-post {{email}}.
+
+Følg denne lenka for å tilbakestille passordet:
```

Comment:
For meg er «linken» et mer innarbeidet ord om en hyperlenke, men det kan hende at det er vestlandsbakgrunnen min som snakker 🤷‍♂️

Reply:
Jeg er fan av norvagisering, så jeg brukte "lenka". Men enig i at "linken" sikkert er bedre. Vi må bestemme oss for en språkstil og så prøve å bruke den konsekvent. Generelt sett er all teksten jeg har fylt inn kun skjelett, så det er bare å endre på rettskriving der dere synes det blir for gæli!

Reply:
Jeg bruker mye hunkjønn også, ser du. Vet ikke hvilken linje vi skal legge oss på der.

Reply:
Vi kan gjerne bruke mye hukjønn så lenge vi er konsekvente. I så fall kommer jeg til å trenge hjelp med det :)